docs/06_how-to-guides/30_key-value-api/kv_table/20_how-to-create-indexes-kv-table.md

```diff
@@ -80,13 +80,14 @@ class [[eosio::contract]] smrtcontract : public contract {
 };
 ```
 
-### Define a unique index on property personal_id using the eosio::kv::table::index template class
+### Define a unique index on data member personal_id using the eosio::kv::table::index template class
```

Comment:
Define a unique index on =>the data member personal_id using the eosio::kv::table::index template class